NativeApp/src/components/Booking/container.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import { Alert } from 'react-native';
+import { PropTypes as PT } from 'prop-types';
+import moment from 'moment';
+import { userProfile } from '../../utilities/currentUser';
+import { requestHolidays } from '../../services/holidayService';
+
+
+export default Container => class extends Component {
+  static propTypes = {
+    navigation: PT.shape({
+      navigate: PT.fun,
```

Comment:
Not sure you need to go this deep with propTypes. Especially since the navigation property is coming from a third party library and isn't something we can control.

But if you want to keep this, I think this should be `navigate: PT.func`?

Reply:
Linter was complaining about this, forbids vague prop types. I can set the linter to ignore this but think I prefer keeping it in. What do you think?

Reply:
I think we shouldn't be using AirBnB's linting rules for a start 🙃 

But that's just me, keeping it in and fixing the `PT.func` is grand!